blockchain/api.go

```diff
@@ -49,7 +51,7 @@ type ProfileRegistryAPI interface {
 }
 
 type EventsAPI interface {
-	GetEvents(ctx context.Context, fromBlockInitial *big.Int) (chan *Event, error)
+	GetEvents(ctx context.Context, filter *EventFilter, fromBlockInitial *big.Int) (chan *Event, error)
```

Comment:
maybe also move fromBlock into filters

Reply:
good idea for next fix.

Reply:
next fix, seriously? You're just peretrakhal all of the filters, why not just peretrakhat' this one?

Reply:
you inspired me to do this =)